include/s1ap/s1ap_macros.h

```diff
@@ -0,0 +1,36 @@
+/*
+* Copyright (c) 2003-2020, Great Software Laboratory Pvt. Ltd.
+*
+* SPDX-License-Identifier: Apache-2.0
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*  http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+
+#ifndef __S1AP_MACROS_H_
```

Comment:
Is it possible to keep the define values in a file which has name with the spec number..e.g. 3gpp_s1ap_36413.h something like this ?  

And in this file keep only #define coming from this spec... Do the same thing for other spec ?